src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -6,8 +6,9 @@
 from collections import OrderedDict
 
 _property_map = {
-    'name': 'NAME',
-    'resourceGroup': 'RESOURCE GROUP',
+    #'name': 'NAME',
+    #'resourceGroup': 'RESOURCE GROUP',
+    'buildType': 'TYPE',
```

Comment:
I would recommend `BUILD TYPE` here as `TYPE` conflicts with ARM resource type.

Reply:
Update: we are going to have "TASK" for that right now. How does that sound?